data_model/src/block.rs

```diff
@@ -166,6 +166,7 @@ impl SignedBlock {
 
     /// Signatures of peers which approved this block.
     #[inline]
+    #[allow(private_interfaces)]
     pub fn signatures(&self) -> &SignaturesOf<BlockPayload> {
```

Comment:
this is a bit suspicious looking. But I still don't think `BlockPayload` should be public